.gitignore

```diff
@@ -18,4 +18,5 @@ robot_gui/reports/
 secrets.json
 setups.json
 log/
-virenv/
\ No newline at end of file
+virenv/
+secrets.json
```

Comment:
why adding twice @seagate-sarang-sawant ? I can see it at line number 18 as well

Reply:
I see. This one is old PR. Some other PRs must have added that.